gradle/check-licenses.gradle

```diff
@@ -169,6 +169,7 @@ downloadLicenses {
       //JMH-Core is licensed under GPLv2 with the Classpath Exception, which allows us to link it and license the derived work under our license.
       'org.openjdk.jmh:jmh-core:1.21': apache,
       (group('io.libp2p')): apache,
+      (group('tech.pegasys')): apache,
```

Comment:
We should probably log a ticket to get the license information included in the blst native jar.  I can't remember the magic incantation and its not a priority but something we should do when we setup a full CI for it and then we shouldn't need this.